esp/esp/web/util/latex.py

```diff
@@ -140,59 +140,40 @@ def gen_latex(texcode, type='pdf', landscape=False, remove_files=False, stdout=P
     
     #   Set latex options
     latex_options = ['-interaction', 'nonstopmode', '-halt-on-error']
-    #   Set dvips options
-    dvips_options = ['-t', 'letter']
-    if landscape:
-        dvips_options = ['-t', 'letter,landscape']
 
     # All command calls will use the same values for the cwd, stdout, and
     # stderr arguments, so we define a partially-applied callable call()
     # that makes it easier to call check_call() with these values.
     call = partial(check_call, cwd=TEX_TEMP, stdout=stdout, stderr=stderr)
 
-    if type=='pdf':
+    if type == 'pdf':
         mime = 'application/pdf'
-        call(['latex'] + latex_options + ['%s.tex' % file_base])
-        call(['dvips'] + dvips_options + ['%s.dvi' % file_base])
-        call(['ps2pdf', '%s.ps' % file_base])
-        if remove_files:
-            os.remove('%s.dvi' % file_base)
-            os.remove('%s.ps' % file_base)
-            
-    elif type=='dvi':
-        mime = 'application/x-dvi'
-        call(['latex'] + latex_options + ['%s.tex' % file_base])
-        
-    elif type=='ps':
-        mime = 'application/postscript'
-        call(['latex'] + latex_options + ['%s.tex' % file_base])
-        call(['dvips'] + dvips_options + [file_base, '-o', '%s.ps' % file_base])
-        if remove_files:
-            os.remove('%s.dvi' % file_base)
-        
-    elif type=='log':
+        call(['pdflatex'] + latex_options + ['%s.tex' % file_base])
+
+    elif type == 'dvi':
```

Comment:
And you can remove these from ``file_types`` and delete these cases, just relying on the ``else`` at the end.